codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieEncoder.java

```diff
@@ -68,6 +68,7 @@ public static String encode(Cookie cookie) {
 
         if (cookie.maxAge() != Long.MIN_VALUE) {
             add(buf, CookieHeaderNames.MAX_AGE, cookie.maxAge());
+            add(buf, CookieHeaderNames.EXPIRES, HttpHeaderDateFormat.get().format(cookie.maxAge() * 1000 + System.currentTimeMillis()));
```

Comment:
Thanks.  Just wanted to make sure the conversion was being done some where...is the conversion in `HttpHeaderDateFormat.format`?